bin/sweepj2

```diff
@@ -55,3 +58,6 @@ for comb in itertools.product(*vals):
     outfile = "{}/{}-{}{}".format(args['output'], str(count).zfill(3), outfile, ext)
```

Comment:
can we also include the name of the template without extension. In the example this would generate `script-008-20-0.5-3`, etc..